pkg/virt-controller/watch/pool/pool_test.go

```diff
@@ -635,55 +587,56 @@ var _ = Describe("Pool", func() {
 			controller.vmIndexer.Add(vm)
 			addPool(pool)
 
-			expectVMCreation(HavePrefix(fmt.Sprintf("%s-", pool.Name)))
-
-			fakeVirtClient.Fake.PrependReactor("update", "virtualmachinepools", func(action k8stesting.Action) (handled bool, obj runtime.Object, err error) {
-				update, ok := action.(k8stesting.UpdateAction)
-				Expect(ok).To(BeTrue())
-				updateObj := update.GetObject().(*poolv1.VirtualMachinePool)
-				Expect(updateObj.Status.Replicas).To(Equal(int32(0)))
-				return true, update.GetObject(), nil
-			})
-
 			poolRevision := createPoolRevision(pool)
-			expectControllerRevisionCreation(poolRevision)
+
+			createVMsWithOrdinal(pool, 3, poolRevision, poolRevision, vm)
 
 			sanityExecute()
-			testutils.ExpectEvent(recorder, common.SuccessfulCreateVirtualMachineReason)
-			testutils.ExpectEvent(recorder, common.SuccessfulCreateVirtualMachineReason)
 
+			cr, err := k8sClient.AppsV1().ControllerRevisions(pool.Namespace).Get(context.TODO(), poolRevision.Name, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(cr.Name).To(Equal(poolRevision.Name))
```

Comment:
Do we have to test in every case that a CR was created? IMO it is enough to test it in one case that specifically tests that behavior?